addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
Well, I'm impressed with @sandersky's work here, but I share @psbanka's concerns.  That said, I think we can use this for now, but I am wondering if there's another way to side-load this other than direct inheritance.  Is a mixin worth considering here? (not a huge fan of those, but it might be more the "ember way" here).  Also, I would really like to see this work submitted as a PR against ember core directly, regardless of what we do with it here.  Thoughts?